modules/sum-transformers/additional/summary/summary_params_extractor.go

```diff
@@ -0,0 +1,37 @@
+//                           _       _
+// __      _____  __ ___   ___  __ _| |_ ___
+// \ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+//  \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+//   \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+//
+//  Copyright © 2016 - 2022 SeMI Technologies B.V. All rights reserved.
+//
+//  CONTACT: hello@semi.technology
+//
+
+package summary
+
+import (
+	"github.com/graphql-go/graphql/language/ast"
+)
+
+func (p *SummaryProvider) parseSummaryArguments(args []*ast.Argument) *Params {
+	out := &Params{}
+
+	for _, arg := range args {
+		switch arg.Name.Value {
+		case "properties":
+			inp := arg.Value.GetValue().([]ast.Value)
+			out.Properties = make([]string, len(inp))
+
+			for i, value := range inp {
+				out.Properties[i] = value.(*ast.StringValue).Value
+			}
+
+		default:
+			// ignore what we don't recognize
```

Comment:
I am not sure about this but at least the non recognised parameters need to be logged

Reply:
Done